core/src/main/scala-3/spire/std/double.scala

```diff
@@ -0,0 +1,169 @@
+/*
+ * **********************************************************************\
+ * * Project                                                              **
+ * *       ______  ______   __    ______    ____                          **
+ * *      / ____/ / __  /  / /   / __  /   / __/     (c) 2011-2021        **
+ * *     / /__   / /_/ /  / /   / /_/ /   / /_                            **
+ * *    /___  / / ____/  / /   / __  /   / __/   Erik Osheim, Tom Switzer **
+ * *   ____/ / / /      / /   / / | |   / /__                             **
+ * *  /_____/ /_/      /_/   /_/  |_|  /____/     All rights reserved.    **
+ * *                                                                      **
+ * *      Redistribution and use permitted under the MIT license.         **
+ * *                                                                      **
+ * \***********************************************************************
+ */
+
+package spire
+package std
+
+import spire.algebra.{Field, IsRational, NRoot, Order, Signed, Trig, TruncatedDivisionCRing}
+import spire.math.Rational
+import spire.util.Opt
+
+import java.lang.Math
+
+trait DoubleIsField extends Field[Double] {
+  override inline def minus(a: Double, b: Double): Double = a - b
+  inline def negate(a: Double): Double = -a
+  inline def one: Double = 1.0
+  inline def plus(a: Double, b: Double): Double = a + b
+  override inline def pow(a: Double, b: Int): Double = Math.pow(a, b)
+  override inline def times(a: Double, b: Double): Double = a * b
+  inline def zero: Double = 0.0
+
+  override def fromInt(n: Int): Double = n
+
+  override def fromDouble(n: Double): Double = n
+  inline def div(a: Double, b: Double): Double = a / b
+}
```

Comment:
Is this the only code that changed? If so, can we avoid duplicating all the code below by leaving it under `scala`?

Reply:
I also am planning on doing the rest of the methods (except conversions)

Reply:
Also at least for Int, Long and Float

Reply:
I don't think I need to duplicate `DoubleAlgebra` and `DoubleInstances` however, I could move those back under `scala`

Reply:
I am being kind of iterative and experimental on this first one, since the full CI sometimes catches things I miss locally.  The others should be faster.